src/GafferImage/ImageTransform.cpp

```diff
@@ -216,9 +316,16 @@ void ImageTransform::hash( const ValuePlug *output, const Context *context, IECo
 
 	if( output == resampleMatrixPlug() )
 	{
-		transformPlug()->translatePlug()->hash( h );
-		transformPlug()->scalePlug()->hash( h );
-		transformPlug()->pivotPlug()->hash( h );
+		transformPlug()->hash( h );
+		inTransformPlug()->hash( h );
+	}
+	else if( output == outTransformPlug() )
+	{
+		inTransformPlug()->hash( h );
```

Comment:
Because it's not necessary, and because the general rule in Gaffer is that nodes should be blissfully ignorant about their connection to the outside world. We have to bend the rules to manage the inTransform connection, but there's no need to do so here.